AUTHORS.txt

```diff
@@ -1,4 +1,4 @@
-# ACE3 CONTRIBUTOR LIST
+﻿# ACE3 CONTRIBUTOR LIST
```

Comment:
BOM added, `0xEF0xBB0xBF` which can't be seen

https://en.wikipedia.org/wiki/Byte_order_mark